README.md

```diff
@@ -17,16 +17,47 @@ Python 3.x installed, you may need to suffix all `python` and `pip` commands
 with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
 3 your default Python installation.
 
+
 ## Cloning and running it locally
 
-1. Clone the project (`git clone https://github.com/codethesaurus/codethesaur.us.git`)
+1. Clone the project (`C`)
 1. Install [Python](https://www.python.org/downloads/)
 1. For ease of use, make sure your Python folder is in your `PATH` environment variable
 1. Run `pip install -r requirements.txt`
 1. Then run `python manage.py runserver`
 1. In your browser, visit [http://127.0.0.1:8000/](http://127.0.0.1:8000/) or [http://localhost:8000/](http://localhost:8000/)
 1. Press CTRL+C in the terminal to stop the server.
 
+## For mac users
+
+- open your terminal, clone the project
+```
+git clone https://github.com/codethesaurus/codethesaur.us.git
+
+pip3 install virtualenv
+
+// set up new virtual environment
+
+virtualenv --no-site-packages venv
+
+// activate virtual environment
+
+source venv/bin/activate
+
+pip install -r requirements.txt
+
+python manage.py runserver
+
+//  In your browser, visit http:localhost:8000
+//   CTRL - C in the terminal to stop the server
+
+// deactivate the virtual environment
+
+deactivate
+```
```

Comment:
These directions are great, but they'd be a bit easier to parse if it wasn't a large code block with directions and things to type in at the same time. Could you separate them out into list items like above (lines 23-29)?

Reply:
Hi I have updated it to keep format consistency, please check if it satisfies. Thanks!